src/Tribe/REST/V1/Post_Repository.php

```diff
@@ -511,7 +543,7 @@ public function prepare_term_data( $term_data, $taxonomy, $namespace ) {
 
 		unset( $term_data['_links'] );
 
-		$term_id = Tribe__Utils__Array::get( $term_data, 'id', $term_data['term_id'] );
+		$term_id = isset( $term_data['id'] ) ? $term_data['id'] : $term_data['term_id'];
```

Comment:
After the `foreach ( $rename_map as $old => $new )` above, this _seems_ redundant?